### PR TITLE
Add splash screen

### PR DESCRIPTION
**Pull request by IstrajI**

### CODE REVIEW

File: ui/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -72,4 +72,9 @@
         <item name="android:textSize">14sp</item>
     </style>
 
+    <!-- Splash Window -->
+    <style name="SplashTheme" parent="Theme.AppCompat.NoActionBar">
+        <item name="android:windowBackground">@drawable/background_splash</item>
+    </style>
+
```
Comment (mKantsavoi) [inline ui/src/main/res/values/styles.xml]: Theme must be in themes file

Comment (mKantsavoi) [reply ui/src/main/res/values/styles.xml]: And maybe better change parent to our Popcorn Theme

**Review by IstrajI — COMMENTED** [review]
test

File: ui/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -72,4 +72,9 @@
         <item name="android:textSize">14sp</item>
     </style>
 
+    <!-- Splash Window -->
```
Comment (mKantsavoi) [inline ui/src/main/res/values/styles.xml]: Splash Theme

File: app/src/main/java/com/blaizmiko/popcornapp/application/BaseApplication.java [hunk 3]
```diff
@@ -29,7 +29,7 @@ private void initCalligraphy() {
                 .build());
     }
 
-    //Returns retrpfit Object for BASE_URL
+    //Returns retofit Object for BASE_URL
```
Comment (mKantsavoi) [inline app/src/main/java/com/blaizmiko/popcornapp/application/BaseApplication.java]: Attempt number 1. Try again

**Pull request merged**